Makefile

```diff
@@ -1,4 +1,5 @@
 SERVER_URL = http://localhost:8000
```

Comment:
Also, we have theorically different targets which are defined by [configuration files](https://github.com/mozilla-services/loop-server/tree/master/loadtests/config), we should continue to do the same thing I believe :)